Tests/DecodableTests.swift

```diff
@@ -165,7 +165,7 @@ class DecodableTestCase: BaseTestCase {
 
         // When
         do {
-            let _ = try String(json: json)
+            let _ = try String(json: json as AnyObject)
```

Comment:
These are all kinda nasty. Rather than typecasting here...we should probably cast the `json` at the point of declaration.

```swift
let json: AnyObject = ["key": 10]
```

Does it actually work that way?